.eslintrc.json

```diff
@@ -24,7 +24,7 @@
     "@typescript-eslint/semi": "error",
     "brace-style": "off",
     "func-call-spacing": "off",
-    "max-len": ["warn", { "code": 100 }],
+    "max-len": ["warn", { "code": 100, "ignoreStrings": true }],
```

Comment:
Instead of overriding the rule, break the string over multiple lines instead to maintain readability.

e.g.

```
const string = `my really long\
 string`;
```

Reply:
Is that what we want?  https://github.com/airbnb/javascript#strings--line-length

I was reading airbnb's javascript style guide suggest not breaking it up to that it's more searchable.  I don't really care one way or the other but I really dislike string concat.

Reply:
That works for me and I feel the 'painful to work with' argument despite the reduced readability.

[updated the code style guide to reflect this new guidance.](https://github.com/mcode/pathways/wiki/Code-Style-Guide#strings)